<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-8L0GRH4V" failures="1" tests="6" name="Test1" time="0.053" errors="0" timestamp="2020-08-18T23:22:23 EDT">
  <testcase classname="test.Basics" name="Demo1" time="0.004"/>
  <testcase classname="test.SecCl" name="Demo" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.SecCl.Demo(SecCl.java:28)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.SecCl" name="Demo1" time="0.001"/>
  <testcase classname="test.SecCl" name="dataTest" time="0.001"/>
  <testcase classname="test.SecCl" name="dataTest" time="0.001"/>
  <testcase classname="test.SecCl" name="dataTest" time="0.001"/>
  <testcase classname="test.Basics" name="Demo" time="0.0">
    <ignored/>
  </testcase> <!-- Demo -->
</testsuite> <!-- Test1 -->
